src/runtime/HalideRuntimeHexagonHost.h

```diff
@@ -83,12 +86,12 @@ typedef struct {
     int latency;
 } halide_hvx_power_perf_t;
 
+/** Set a performance target for HVX. HVX applications can vote for
+ * the performance levels they want, which may or may not be respected
+ * by Hexagon. */
 // @{
-extern int halide_hexagon_power_hvx_on(void *user_context);
-extern int halide_hexagon_power_hvx_on_mode(void *user_context, halide_hvx_power_mode_t mode);
-extern int halide_hexagon_power_hvx_on_perf(void *user_context, halide_hvx_power_perf_t *perf);
-extern int halide_hexagon_power_hvx_off(void *user_context);
-extern void halide_hexagon_power_hvx_off_as_destructor(void *user_context, void * /* obj */);
+extern int halide_hexagon_set_performance_mode(void *user_context, halide_hvx_power_mode_t mode);
+extern int halide_hexagon_set_performance(void *user_context, halide_hvx_power_perf_t *perf);
```

Comment:
Needs to be halide_hexagon_set_performance_perf...or the symbol in hexagon_host.cpp needs to be changed to halide_hexagon_set_performance (no preference)